fix(rdb): new troubleshooting pages (#4315)
scaleway/docs-content: pages/managed-databases-for-postgresql-and-mysql/troubleshooting/database-instance-unavailable.mdx

  description: Troubleshoot Database Instance unavailability for Managed Databases for MySQL and PostgreSQL.
content:
  h1: Dealing with Database Instance unavailability
  paragraph: Troubleshoot Database Instance unavailability for Managed Databases for MySQL and PostgreSQL.
tags: disk-full databases
dates:
  validation: 2025-01-29
  posted: 2025-01-29
categories:
  - managed-databases
  - postgresql-and-mysql
---

## Problem

My Database Instance is unavailable.

## Possible causes

- High data loads
- High incoming traffic from specific apps or websites

## Solution

You can monitor your Database Instance activity to be able to quickly identify and act on irregular activities that might be causing the instability/unavailability.

[Monitor usage metrics on Cockpit](/managed-databases-for-postgresql-and-mysql/how-to/monitor-databases-cockpit) to:

    - Check if the usage is regular. Database Instance can be impacted by high data loads or unexpected high traffic from an app or website, for example
    - Check if there are slow queries in the Database Instance log. Huge or sub-optimized queries can fill up all the memory and trigger OOM kills

>     - Check Database Instance logs and look for anything else that could explain the issue

Reviewer: is it possible to be more specific here @bastienwirtz ? key words to look in the logs maybe or type of logs that we often see in those cases?

Reviewer 2: We can rephrase it a little maybe but I don't really have any specifics in mind, the idea was more a general check, but that could be better : 

```suggestion
    - Check Database Instance logs for anything unusual or suspicious.
```
